lib/sport_ngin_aws_auditor/scripts/audit.rb

```diff
@@ -52,26 +54,36 @@ def self.gather_data(class_type, tag_name)
           instance_hash = klass.instance_count_hash(klass.get_reserved_instances)
         else
           instance_hash = klass.compare(tag_name)
+          retired_reserved_instances = klass.get_recent_retired_reserved_instances
         end
 
+        return_array = []
         compared_array = []
+
         instance_hash.each do |key, value|
           compared_array.push(Instance.new(key, value))
         end
-        compared_array
+
+        return_array.push(compared_array)
+        return_array.push(retired_reserved_instances) if retired_reserved_instances
+        return_array
```

Comment:
you could do this in one line if you do `[comared_array, retired_reserved_instances]`